ravendb/rvn_server.go

```diff
@@ -1083,7 +1091,7 @@ func (sc *ServerConfig) modifyDatabase(store *ravendb.DocumentStore, dbRecord *r
 		if rmDbInTags != nil {
 			operation := ravendb.NewDeleteDatabasesOperationWithParameters(&ravendb.DeleteDatabaseParameters{
 				DatabaseNames:             []string{dbRecord.DatabaseName},
-				HardDelete:                false,
+				HardDelete:                true,
```

Comment:
Shouldn't this be coming from the user?

Reply:
It's the situation after changes in `replication_nodes`.
e.g deploying on `[ "A" , "B", "C" ]` and we save the new configuration with only `"A"`
Is this the wrong way to handle it ?